### DISCUSSION TITLE
Image store

### DISCUSSION THREAD

**Discussion by GoBigorGoHome** [edemaine/coauthor#616] [Q&A] [answered]
I would like to store images used in messages inside coauthor. One way is to "Add File" to a message, but then the image will be displayed at two places, one inside the message, another at  bottom of the message. Another way I think of is to have a separate group and create messages there, "Add File"s to them, and put a link in messages where images are actually used. But I have not tried.  

Is there any other ways to store images inside the coauthor system?

**Comment by edemaine** [comment, marked as answer]
If you attach an image as a separate message (e.g. a reply), and then include that message in another post (e.g. parent), then the image file will be minimized. This is the "right" way to do it currently.

**Reply by GoBigorGoHome** [reply]
Thank you! I didn't notice the "Reply with Attached File" button.

**Reply by edemaine** [reply]
Add File is actually new, and it was mofe obvious before that. We should consider making Add File less prominent...
